protocol-units/settlement/mcr/client/src/eth_client.rs

```diff
@@ -65,6 +82,7 @@ pub struct Client<P> {
 	send_tx_error_rules: Vec<Box<dyn VerifyRule>>,
 	gas_limit: u128,
 	num_tx_send_retries: u32,
+	kill_anvil_process: Option<AnvilKillAtDrop>,
```

Comment:
I don't think this belongs in the client.

In the production use case, we don't run Anvil, so let's not make this a wart on `Client`, but compose this helper with `Client` somewhere in test infrastructure?

Reply:
Yes it's a temporary solution until process management is settlement. Currently I've no other way to indicate a process from one process to the other.